lib/smack/SmackRep.cpp

```diff
@@ -758,36 +1027,103 @@ const Expr* SmackRep::declareIsExternal(const Expr* e) {
 }
 
 string SmackRep::getPtrType() {
-  return "int";
+  if (BIT_VECTOR)
+    return "ref";
+  else
+    return "int";
+}
+
+string SmackRep::getByteType() {
```

Comment:
Probably should remove this, preferring the use of `int_type(8)` in place of `getByteType()`.